### ISSUE TITLE
[RC12]: Urgent: Constantly loud alarms: using AAPS is almost no longer possible

### ISSUE THREAD

**Issue by thomasmenges** [MilosKozak/AndroidAPS#1489]
I have a strange problem in AAPS [RC12].
Sometimes (especially when restarting AAPS /  the smartphone) a loud alarm is attempting which I have to mute before AAPS is starting. Sometimes the problem also exists during the day (not very helpful at work).

The message is always the same:
A treatment (insulin: xxxx, carbs: yyy at: mm/dd/yy time) could not be added to treatments. Please check and manually add a record as appropriate.

The shown treatments  (sometimes more than one) are always from the past (beginning of 2018/10/02).
I didn't change something at this date.

To solve the problem I did a lot of things. 

1) I added a treatment manually at the shown time and date. No sucess. The problem still exists. (Other announcements are shown).
2) I reset the DB in AAPS --> No success. Problem still exists.
3) renew treatment of NS --> No success. Problem still exists.
4) Deinstallation of AAPS an new creating the APP --> No success

Now I'm a little bit frustrated because I do not know what I can do to solve the problem.

As attachments I have included the last logs and a screenshot of the treatment overview in AAPS
![aaps_problems_treatment_saving1](https://user-images.githubusercontent.com/37514390/46521770-ed746f00-c880-11e8-96ad-b71d90dcea96.jpg)

[AndroidAPS._2018-10-05_08-56-15_.0.txt](https://github.com/MilosKozak/AndroidAPS/files/2449456/AndroidAPS._2018-10-05_08-56-15_.0.txt)

[AndroidAPS._2018-10-05_08-13-13_.0.txt](https://github.com/MilosKozak/AndroidAPS/files/2449458/AndroidAPS._2018-10-05_08-13-13_.0.txt)

What can I do to solve the problem ?

**Comment by thomasmenges** [comment]
Sorry: The same problem is mentioned in #1482. Have not seen it before.

**Comment by MilosKozak** [comment]
i need time when this happened. it's not possible to read whole logfile

**Comment by thomasmenges** [comment]
Yes that's right. But if I restart  AAPS other treatments are mentioned. Right now it is
The announcement from yesterday of the new screenshot and Logfile
[AndroidAPS._2018-10-07_12-45-15_.0.zip](https://github.com/MilosKozak/AndroidAPS/files/2453822/AndroidAPS._2018-10-07_12-45-15_.0.zip)
![20181007_124652](https://user-images.githubusercontent.com/37514390/46581011-58b07380-ca30-11e8-9351-10360efe06f7.jpg)

**Comment by MilosKozak** [comment]
hmmm it's Insight, I see now
Insight support is very exprimental and driver will be replaced by another one

**Comment by thomasmenges** [comment]
Hello Milos,
no I'm using a combo and not an Insight !!!!

**Comment by MilosKozak** [comment]
ooops sorry.
@jotomo could you have a look?

**Comment by thomasmenges** [comment]
@MilosKozak Sorry for my delay. I was a week at a conference and could not test your solution before. I have installed RC14 and the problem is solved completly. Thanks.
